connector/signaltometricsconnector/README.md

```diff
@@ -0,0 +1,3 @@
+# Signal to metrics connector
+
+WIP
```

Comment:
[**For reviewers**] I will update this in a follow-up after all the configuration changes are in.